packages/client/src/clients/feed/feed.ts

```diff
@@ -667,6 +674,42 @@ class Feed {
       console.warn(`Could not broadcast ${type}, got error: ${e}`);
     }
   }
+
+  /**
+   * Listen for changes to document visibility and automatically disconnect
+   * or reconnect the socket after a delay
+   */
+  private setupAutoSocketManager(
+    auto_manage_socket_connection: boolean | number,
```

Comment:
Nit: I know we have this as snake cased as the option name, but internally here I would expect it as camelCase